recipes/logr/0.2.0/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
I guess 3.14 for supporting c++17?

Reply:
I use it as the lowest version with VS2019 generator. And in general I try to use +/- fresh cmake.

Reply:
I understand your motives, but I don't think this recipe uses any new cmake features.
So I guess it's ok to lower it to a version supporting c++17: 3.8 (https://cmake.org/cmake/help/v3.8/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD)

Reply:
Yep, that is reasonable. Is it necessary for projects own cmake?